src/SFML/Window/Unix/WindowImplX11.cpp

```diff
@@ -1966,6 +2069,106 @@ bool WindowImplX11::processEvent(XEvent& windowEvent)
     return true;
 }
 
+
+////////////////////////////////////////////////////////////
+bool WindowImplX11::checkXRandR(int& xRandRMajor, int& xRandRMinor)
+{
+    // Check if the XRandR extension is present
+    int version;
+    if (!XQueryExtension(m_display, "RANDR", &version, &version, &version))
+    {
+        err() << "XRandR extension is not supported" << std::endl;
+        return false;
+    }
+
+    // Check XRandR version, 1.2 required
+    if (!XRRQueryVersion(m_display, &xRandRMajor, &xRandRMinor) || xRandRMajor < 1 || (xRandRMajor == 1 && xRandRMinor < 2 ))
+    {
+        err() << "XRandR is to old" << std::endl;
```

Comment:
too* I guess?

Reply:
You guessed right :)
I have pushed the fix.